modules/dataquery/help/dataquery.md

```diff
@@ -1,44 +1,83 @@
 
-## Permission
+If you click on any category, its field names and descriptions will appear in a table.<br>
 
-The DQT requires the `dataquery_view` permission.
+If the number of fields in the chosen category exceeds the display limit per page, the results may be displayed on subsequent pages within the table. Navigating using the page number buttons toward the bottom of the page. <br>
 
-## Configuration
+Any fields that are defined for the current query are listed on the right-hand side (subject to change). If there are no fields defined for the current query, this list is not shown. Time points can be specified within the field in the **Define Fields** tab or in the **Define Filters** tab.<br>
 
-CouchDB must be independently setup, and the relevant CouchDB views
-installed following the directions at
-https://github.com/aces/Data-Query-Tool.
+The options to add fields to your query are as follows:<br>
 
-Importer scripts must be set up to import data into CouchDB. (The
-instructions are also in the README of the above repository.)
+**Option 1: Add Using Dropdown Fields** - Seach for fields using the list of categories. After clicking the desired instrument, a list of fields are displayed. Click on a field row to highlight it and add it to the list of selected fields for your current query. <br>
 
-## Interactions with LORIS
+**Option 2: Search Within Instrument Fields** - Search for fields using the search bar. Once you start typing, all matches will be displayed in an active list. Click the desired field to add it to the query. <br>
 
-The CouchDB import scripts (those named `CouchDB_Import_*.php` in
-the LORIS tools directory) must be run to import data from MySQL
-to CouchDB before the DQT can be used.
+**Option 3: Add All** - Use this option to add all fields from a category to the query. To do so, click **Add All** above the list of categories. <br>
 
+**Option 4: Load fields from a previously saved query** - Click on the **Load Saved Query** tab. User Saved Queries and Shared Saved Queries will appear in a dropdown. (*User Saved Queries* are queries that you have previously saved, and are listed according to Query Name. *Shared Saved Queries* are publicly shared, and are listed according to the user who created it and the Query Name.) <br>
+
+Once a query is selected, the DQT will retrieve and load both the fields and filters defined in the saved query, displayed in the **Define Fields** and **Define Filters** tabs. To remove fields from your query, find the field from the list (using the Search Within Instrument search bar or in the instrument list of fields) and click on the highlighted field, to de-select it. Confirm its removal by checking that the field no longer appears in your list of selected fields. <br>
+
+To remove *all* currently selected fields, click **Clear Query** in the list of Fields.<br>
```

Comment:
I still see some `<br>` tags, but not sure they need to be removed in this context.